internal/quickstart/messages.go

```diff
@@ -240,3 +241,13 @@ func selectedSDK(index int) tea.Cmd {
 		return selectedSDKMsg{index: index}
 	}
 }
+
+type eventTrackedMsg struct{}
+
+func trackSetupStartedEvent(tracker analytics.Tracker, accessToken, baseURI string, optOut bool, step string) tea.Cmd {
+	return func() tea.Msg {
+		tracker.SendSetupStartedEvent(accessToken, baseURI, optOut, step)
+
+		return eventTrackedMsg{}
+	}
+}
```

Comment:
It's a bit confusing but according to the cli tracking spec. The event "CLI Setup Started" should be called for every step. The Step string will indicate what step we're on.

Reply:
It would be worth clarifying the event name for subsequent steps since it is confusing.

Reply:
@bwoskow-ld Does this sound right? From the spec it looked like "CLI Setup Started" was what would be used on every step. Just with the step string itself denoting what step we're on.

Reply:
Oh yeah that is confusing. Let's rename it to `CLI Setup Step Started` since that's more in line with what this data represents. I'll update the spec page.